doc/examples/streamline_length.py

```diff
@@ -8,17 +8,18 @@
 lengths or overall shape.
 
 A streamline in DIPY_ is represented as a numpy array of size
-:math:`(N \times 3)` where each row of the array represent a 3D point of the
+:math:`(N \times 3)` where each row of the array represents a 3D point of the
 streamline. A set of streamlines is represented with a list of
-numpy arrays of size :math:`(N_i \times 3)` for :math:`i=1:M` where $M$ is the
+numpy arrays of size :math:`(N_i \times 3)` for :math:`i=1:M` where 'M' is the
```

Comment:
Please check the rst syntax. `M` represents a math variable.

Reply:
Thank you for correcting about the math variable. I have changed it other changes still seem valid.